Assets/_Functionalities/Wms/Scripts/WmsGetCapabilities.cs

```diff
@@ -117,17 +128,26 @@ private BoundingBox ParseBoundingBox(XmlNode node, CoordinateSystem crs)
             if (node == null)
                 return null;
 
-            var minXString = node.SelectSingleNode("*[local-name()='westBoundLongitude' or @minx]", namespaceManager)?.InnerText;
-            var minYString = node.SelectSingleNode("*[local-name()='southBoundLatitude' or @miny]", namespaceManager)?.InnerText;
-            var maxXString = node.SelectSingleNode("*[local-name()='eastBoundLongitude' or @maxx]", namespaceManager)?.InnerText;
-            var maxYString = node.SelectSingleNode("*[local-name()='northBoundLatitude' or @maxy]", namespaceManager)?.InnerText;
-
-            //replace , with . to ensure the parse function works as intended.
+            // This method seems to be reused in reading the EX_GeographicBoundingBox -which is always in CRS:84 but I
+            // hope the caller does this correct-
+            var minXString = node.SelectSingleNode("*[local-name()='westBoundLongitude']", namespaceManager)?.InnerText;
+            var minYString = node.SelectSingleNode("*[local-name()='southBoundLatitude']", namespaceManager)?.InnerText;
+            var maxXString = node.SelectSingleNode("*[local-name()='eastBoundLongitude']", namespaceManager)?.InnerText;
+            var maxYString = node.SelectSingleNode("*[local-name()='northBoundLatitude']", namespaceManager)?.InnerText;
+
+            // Ugly solution to support both EX_GeographicBoundingBox and BoundingBox, the latter uses attributes
+            minXString ??= node.Attributes["minx"]?.Value;
+            minYString ??= node.Attributes["miny"]?.Value;
+            maxXString ??= node.Attributes["maxx"]?.Value;
+            maxYString ??= node.Attributes["maxy"]?.Value;
+
+            // replace , with . to ensure the parse function works as intended because some Dutch agencies use the wrong
+            // decimal separators. Because Fuck standards?
```

Comment:
Ik snap je frustratie, maar ik zou dat laatste even weghalen

Reply:
Je hebt gelijk, dat had daar niet in gemoeten. Weggehaald